File: palavras.js
// Lista de palavras para o jogo
const palavras = [
    'PYTHON',
    'JAVASCRIPT',
    'PROGRAMACAO',
    'COMPUTADOR',
    'ALGORITMO',
    'INTERNET',
    'DESENVOLVIMENTO',
    'TECNOLOGIA',
    'SOFTWARE',
    'HARDWARE',
    'DADOS',
    'REDE',
    'SISTEMA',
    'APLICACAO',
    'INTERFACE',
    'BANCO',
    'CODIGO',
    'PROGRAMA',
    'FUNCAO',
    'VARIAVEL'
];

// Função para escolher uma palavra aleatória
function escolherPalavraAleatoria() {
    const indice = Math.floor(Math.random() * palavras.length);
    return palavras[indice];
} 

File: palavras.py
# Lista de palavras organizadas por categorias
PALAVRAS = {
    'tecnologia': [
        'python', 'computador', 'teclado', 'mouse', 'monitor',
        'internet', 'software', 'hardware', 'dados', 'programacao'
    ],
    'animais': [
        'cachorro', 'gato', 'elefante', 'girafa', 'leao',
        'tigre', 'macaco', 'panda', 'coelho', 'tartaruga'
    ],
    'frutas': [
        'banana', 'maca', 'laranja', 'uva', 'morango',
        'abacaxi', 'manga', 'pera', 'kiwi', 'melancia'
    ]
}

# Função para obter todas as palavras de todas as categorias
def obter_todas_palavras():
    todas_palavras = []
    for categoria in PALAVRAS.values():
        todas_palavras.extend(categoria)
    return todas_palavras

# Função para obter palavras de uma categoria específica
def obter_palavras_categoria(categoria):
    return PALAVRAS.get(categoria, []) 